CHANGES.md

```diff
@@ -15,7 +15,11 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+- Makefile: Ensure the `bin/birdhouse` path employed by default resolves from anywhere.
```

Comment:
Can you put a `## Fixes` above this just to keep this in line with the other entries in this file

Reply:
Yes. Well spotted.